Fixes the following error:

    error[E0308]: mismatched types
      --> below/btrfs/src/btrfs_api/sudotest.rs:49:40
       |
    49 |     statfs.filesystem_type() == FsType(libc::BTRFS_SUPER_MAGIC)
       |                                 ------ ^^^^^^^^^^^^^^^^^^^^^^^ expected `u64`, found `i64`
       |                                 |
       |                                 arguments to this struct are incorrect

--- a/below/btrfs/src/btrfs_api/sudotest.rs
+++ b/below/btrfs/src/btrfs_api/sudotest.rs
@@ -46,7 +46,7 @@
         }
     };
 
-    statfs.filesystem_type() == FsType(libc::BTRFS_SUPER_MAGIC)
+    statfs.filesystem_type() == FsType(libc::BTRFS_SUPER_MAGIC.try_into().unwrap())
 }
 
 #[test]
